sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachine.java

```diff
@@ -1765,6 +1768,16 @@ WithSystemAssignedIdentityBasedAccessOrCreate withSystemAssignedIdentityBasedAcc
             WithManagedCreate withAvailabilityZone(AvailabilityZoneId zoneId);
         }
 
+        /** The stage of the VM definition allowing to specify virtual machine scale set */
+        interface WithScaleSet {
+            /**
+             * Specifies an existing virtual machine scale set for the virtual machine.
+             * @param scaleSet the virtual machine scale set with flexible orchestration mode
+             * @return the next stage of the definition
+             */
+            WithManagedCreate withExistingVirtualMachineScaleSet(VirtualMachineScaleSet scaleSet);
```

Comment:
WithManagedCreate or WithCreate  ?

Reply:
WithManagedCreate I believe, I added a assertion in the test: https://github.com/Azure/azure-sdk-for-java/blob/bd048014cd89ee2b89c9bf9749f363a3e40b92dd/sdk/resourcemanager/azure-resourcemanager-compute/src/test/java/com/azure/resourcemanager/compute/VirtualMachineOperationsTests.java#L1377-L1398